MAESTRO/Snakemake/scATAC/Snakefile

```diff
@@ -215,38 +180,36 @@ if config["platform"] == "microfluidic":
         benchmark:
             "Result/Benchmark/%s_AllPeakCall.benchmark" %(config["outprefix"])
         shell:
-            "macs2 callpeak -f BAMPE -g {params.genome} --outdir Result/Analysis/ -n {params.name} -B -q 0.05 --nomodel --extsize=50 --SPMR --keep-dup all -t {input.bam}"
-    
+            "macs2 callpeak -f BEDPE -g {params.genome} --outdir Result/Analysis -n {params.name} -B -q 0.05 --nomodel --extsize=50 --SPMR --keep-dup all -t {input.frag_dedup}"
+
```

Comment:
@taoliu for -f BAMPE or -f BEDPE , --extsize=50 has no effect and can be removed, right?

Reply:
Right